base_business_document_import/models/business_document_import.py

```diff
@@ -292,7 +292,7 @@ def _match_product(
         if product_dict.get('description'):
             config = seller.invoice_import_id
             # Match products
-            for prod in config.product_mapping:
+            for prod in config.product_mapping_ids:
```

Comment:
I don't find the variable `prod` rather indicative. Would you agree to `prod_map`?